test_fixtures/pipelines/experiment.jsonnet

```diff
@@ -0,0 +1,66 @@
+local checkpoint_dir = std.extVar("CHECKPOINT_PATH");
```

Comment:
Shall we rename this file to something like `pipeline_integration_experiment.json` if it is only used in `test_integration.py`?

Reply:
Done.